api_functional_v2_test.go

```diff
@@ -31,6 +31,31 @@ import (
 	"github.com/minio/minio-go/pkg/policy"
 )
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyz01234569"
```

Comment:
is there a reason only a few of the tests are run for v2? We could run all tests for both v2 and v4 in functional_tests.go but it would slow the test run further.